pyccel/codegen/printing/fcode.py

```diff
@@ -206,6 +208,20 @@ def __init__(self, parser, prefix_module = None):
 
         self.prefix_module = prefix_module
 
+    def print_constant_imports(self):
+        """Prints the use line for the constant imports used"""
+        macro = "use ISO_C_BINDING, only: "
```

Comment:
I think we should keep the `intrinsic` tag (see [here](https://riptutorial.com/fortran/example/5650/intrinsic-modules) for details about what this does)